docs/user-guide/tutorials.md

```diff
@@ -44,6 +44,7 @@ following topics:
 | [The Certora Prover pipeline](https://youtu.be/vg6da3A7lSs) | ({download}`slides <basics/stanford/09-pipeline.pdf>`)        |
 | [SMT solvers                ](https://youtu.be/9QuS_8cL91w) | ({download}`slides <basics/stanford/10-smt.pdf>`)             |
 
+The covered examples are available in the [CVL examples repository](https://github.com/Certora/Examples).
```

Comment:
I would link to the specific directory; there's a ton of stuff in the examples repo and it's hard to find the specific ones you're looking for.

Reply:
I linked in the doc to specific examples. This gives a link to the full repository.

Reply:
this link is ok here